app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap

```diff
@@ -225,15 +240,34 @@ exports[`Metadata should render links when the property has url 1`] = `
     <dd
       className=""
     >
-      <Connect(RelationshipLink)
-        prop={
+      <ValueList
+        compact={true}
+        property={
           Object {
             "label": "withUrl",
             "name": "withurl",
             "translateContext": "oneTranslateContext",
             "type": "relationship",
             "url": "url",
-            "value": "string value",
+            "value": Array [
+              Object {
+                "0": "s",
+                "1": "t",
+                "10": "u",
+                "11": "e",
+                "2": "r",
+                "3": "i",
+                "4": "n",
+                "5": "g",
+                "6": " ",
+                "7": "v",
+                "8": "a",
+                "9": "l",
+                "value": <Connect(RelationshipLink)
+                  propValue="string value"
+                />,
+              },
+            ],
```

Comment:
This cannot be correct.  Either you are passing the wrong value to the test, or this means that you are passing each letter of a string like the array values.  Please either fix the text or the implementation.

Reply:
Fixed